pkg/controller/direct/securesourcemanager/mapper.generated.go

```diff
@@ -169,6 +163,7 @@ func SecureSourceManagerInstanceObservedState_ToProto(mapCtx *direct.MapContext,
 	// MISSING: CreateTime
 	// MISSING: UpdateTime
 	// MISSING: Labels
+	// MISSING: PrivateConfig
```

Comment:
I saw the `Instance_PrivateConfig_ToProto` above. Shall we use it here or more work needs to be done for Instance_PrivateConfig_ToProto.